docs/changelog.md

```diff
@@ -5,7 +5,28 @@ Lines for version numbers should always be formatted as
 with nothing else on the line.
 -->
 * HEAD
+* [0.30.0](https://pypi.org/project/sematic/0.30.0/)
+    * [feature] Grafana dashboards tailored for Sematic installable via Helm
+    * [feature] User metrics SDK
+    * [feature] Add horizontal pod autoscaling and pod disruption budget support
+    * [feature] Health endpoint to display DB health in dashboard
+    * [feature] Add support for set types
+    * [feature] Search for runs in dashboard
+    * [feature] Add GitHub commit check support
+    * [feature] Search by tags
```

Comment:
This isn't exposed to users; only the backend implementation went in. It is supporting Chance's work on the new UI.